semcore/inline-edit/src/InlineEdit.tsx

```diff
@@ -114,21 +126,21 @@ const View: React.FC<AsProps & { inlineEditContainerRef: React.RefObject<HTMLEle
   const visible = !props.editable;
   const SView = Root;
 
-  const containerRef = React.useRef<HTMLElement>(null);
-  const prevVisibleRef = React.useRef(visible);
-  React.useEffect(() => {
-    const visibleChanged = prevVisibleRef.current !== visible;
-    if (visible && visibleChanged) {
-      const focusWithinContainer = isElementInside(
-        document.activeElement,
-        props.inlineEditContainerRef.current,
-      );
-      if (focusWithinContainer) {
-        // containerRef.current?.focus();
-      }
-    }
-    prevVisibleRef.current = visible;
-  }, [visible]);
+  // const containerRef = React.useRef<HTMLElement>(null);
```

Comment:
This obsolete code is really not needed any more. Please remove it instead of commenting and PR will be ready for mege